packages/jest-runner/src/run_test.js

```diff
@@ -145,6 +146,28 @@ async function runTestInternal(
   // For runtime errors
   sourcemapSupport.install(sourcemapOptions);
 
+  const realExit = environment.global.process.exit;
```

Comment:
What if there's no `process` in `global`? We'll be calling undefined fn, or am I missing something?

Reply:
it's always there because of the `installCommonGlobals` call. Of course fully custom envs won't necessarily have it...

I can wrap it in an `if`